streams/src/main/java/org/apache/kafka/streams/StreamsConfig.java

```diff
@@ -157,6 +157,7 @@
                                         ConsumerConfig.VALUE_DESERIALIZER_CLASS_DOC)
                                 .define(TIMESTAMP_EXTRACTOR_CLASS_CONFIG,
                                         Type.CLASS,
+                                        WALLCLOCK_TIMESTAMP_EXTRACTOR,
```

Comment:
Sneaky. This has nothing to do with this refactoring.
btw, did we agree on having this as the default timestamp extractor?